content/20.treatments.md

```diff
@@ -540,19 +540,59 @@ Although exciting, the cost of vaccine manufacturing and electroporation may mak
+Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].  
+Furthermore, mRNA viruses are easily, affordably, and rapidly scalable.
+
+mRNA vaccines have been developed for therapeutic and prophylactic purposes. 
+mRNA vaccines have not been licensed or commercialized; however, several agents for cancer and infectious diseases, including rabies, coronavirus, influenza, rabies, and cytomegalovirus, have shown promising immunization in animal models and are currently in clinical trials [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 
+Preclinical data from Pardi et al. identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0]. 
+Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245]. 
+
 
 ##### Anticipated Mechanism
 
-Why it may be useful
+Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape. 
+LNPs can be coated with modalities recognized and engulfed by specific celltypes. 
+LNPs 150nm or less effectively enter into lymphatic vessels. 
+
+There are three types of RNA vaccines: non-replicating, in vivo self-replicating, and in vitro dendritic cell non-replicating [@url:https://www.phgfoundation.org/briefing/rna-vaccines].  
+Non-replicating mRNA vaccines consist of a simple open reading frame (ORF) for the viral antigen flanked by the 5’ UTR and 3’ poly-A tail. 
+In vivo self-replicating vaccines encode a modified viral genome derived from single-stranded, positive sense RNA alphaviruses [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245]. 
+The RNA genome encodes the viral antigen along with proteins of the genome replication machinery, including an RNA polymerase. Structural proteins required for viral assembly are not included in the engineered genome [@doi:10.3389/fimmu.2019.00594]. 
+Self-replicating vaccines produce more viral antigens over a longer period of time, thereby evoking a more robust immune response[@url:https://www.phgfoundation.org/briefing/rna-vaccines].  
+Finally, in vitro dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells. 
+Dendritic cells are potent antigen presenting immune cells that easily take up mRNA and present it to other cells, including B-cells which generate antibodies against the protein. 
```

Comment:
This sentence makes it sound as though DCs present mRNA, but in actuality I think they present peptide fragments. As you mentioned above, after the DCs are transfected ex vivo with the mRNA, their endogenous biosynthetic machinery should translate it into protein. The protein is then cleaved into peptide fragments that are presented on MHCI or MHCII. 

Also, I would suggest rewording the second clause in this sentence. DCs present antigen to T cells, not B cells. B cells can sample the antigen on their own, and are activated by interacting with T cells (ideally the T cells will have been activated by interacting with the DCs).

Antigen presentation in general is quite confusing! Let me know if anything doesn't make sense! :)

Reply:
I updated this section to be a little bit more specific without getting into too many details. How does it look now?

Reply:
It looks perfect!!!! :)